java/build.sh

```diff
@@ -0,0 +1,14 @@
+VERSION="25.02" # Note: The version is updated automatically when ci/release/update-version.sh is invoked
```

Comment:
See comment in `update-version.sh` about inconsistency -- this should probably have a patch version.